app/ui/src/Pages/Projects/components/Project/Project.tsx

```diff
@@ -11,43 +11,68 @@ import { Link } from 'react-router-dom';
 import cx from 'classnames';
 import { formatDate } from 'Utils/format';
 import styles from './Project.module.scss';
+import { ProjectAdmins } from '../../Projects';
+
+type BaseProps = {
+  project: GetProjects_projects;
+};
 
 type Props = {
+  showAdmins: (projectAdmins: ProjectAdmins) => void;
   project: GetProjects_projects;
 };
 
-const Project: FC<Props> = ({ project }) => (
-  <Link
-    to={buildRoute(ROUTE.PROJECT, project.id)}
-    className={cx({
-      [styles.disabled]: project.needAccess || project.archived,
-    })}
-  >
+function Project({ project, showAdmins }: Props) {
+  const disabled = project.needAccess || project.archived;
+  const Component = (
     <div
       className={cx(styles.container, {
         [styles.archived]: project.archived,
       })}
     >
-      <UpperBg project={project} />
+      <UpperBg project={project} showAdmins={showAdmins} />
       <LowerBg project={project} />
       <Band project={project} />
       <Square project={project} />
     </div>
-  </Link>
-);
+  );
+  return disabled ? (
+    <div className={styles.disabled}>{Component}</div>
+  ) : (
+    <Link to={buildRoute(ROUTE.PROJECT, project.id)}>{Component}</Link>
+  );
+}
 
-const UpperBg: FC<Props> = ({ project }) => (
-  <div className={styles.sup}>
-    <div className={styles.bg}>
-      <div className={styles.bgBand} />
-    </div>
-    <div className={styles.content}>
-      <p className={styles.name}>{project.name}</p>
+function UpperBg({ project, showAdmins }: Props) {
+  function onContactAdmins() {
```

Comment:
I really don't know if it is a best practise in React but this function should be named: `handleContactAdmins`:
```suggestion
  function handleContactAdmins() {
```